source/dem/dem.cc

```diff
@@ -546,6 +492,44 @@ DEMSolver<dim>::load_balance()
         << average_minimum_maximum_cells.max << std::endl;
 }
 
+template <int dim>
+inline bool
+DEMSolver<dim>::is_load_balance_step()
+{
+  // Setting load-balance method (single-step, frequent or dynamic)
+  if (parameters.model_parameters.load_balance_method ==
+      Parameters::Lagrangian::ModelParameters::LoadBalanceMethod::once)
+    {
+      return check_load_balance_once();
+    }
+  else if (parameters.model_parameters.load_balance_method ==
+           Parameters::Lagrangian::ModelParameters::LoadBalanceMethod::frequent)
+    {
+      return check_load_balance_frequent();
+    }
+  else if (parameters.model_parameters.load_balance_method ==
+           Parameters::Lagrangian::ModelParameters::LoadBalanceMethod::dynamic)
+    {
+      return check_load_balance_dynamic();
+    }
+  else if (parameters.model_parameters.load_balance_method ==
+           Parameters::Lagrangian::ModelParameters::LoadBalanceMethod::
+             dynamic_with_disabling_contacts)
+    {
+      return check_load_balance_with_disabled_contacts();
+    }
+  else if (parameters.model_parameters.load_balance_method ==
```

Comment:
I think "none" should be the first to be tested.

Reply:
Ok, that's aesthetics but I don't mind.